vnext/ReactUWP/Utils/Helpers.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <stdint.h>
+#include <winrt/Windows.UI.Xaml.h>
+#include <IReactInstance.h>
+#include <folly/dynamic.h>
+
+namespace winrt {
+  using namespace Windows::UI::Xaml;
+}
+
+namespace react { namespace uwp {
+
+  using namespace std;
+
+  struct ReactId {
+    int64_t tag{ 0 };
+    bool isValid{ false };
+  };
+
+  template<typename T>
+  inline typename T as(folly::dynamic const& obj)
+  {
+    return (T)(obj.asInt());
```

Comment:
It's for enum type and primary types. as<EnumType> would do the conversion automatically.
also cover things like uint, short, byte,char.

Do we have case which is outside of this?

Reply:
because javascript only has a type 'number' which is basically a double, all the other types don't really exist with folly.  (well other js primitives - number, string, boolean, array, object are all you can have)

Reply:
I removed them and create an new asEnum template